### ISSUE TITLE
Source code repo is out of date

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#168]
```
The source code repo on code.google.com is stuck at version 1.2. Are there any 
plans to release newer versions of the source?
```

Original issue reported on code.google.com by `goo...@dwilga.info` on 17 May 2012 at 1:36

**Comment by GoogleCodeExporter** [comment]
```
Do you have some code you wish to contribute?
```

Original comment by `chris...@gmail.com` on 21 May 2012 at 3:27

**Comment by GoogleCodeExporter** [comment]
```
Possibly, but first I just want to see if it's any faster when compiled as a 
64-bit app.

I also assume the reason I can't get the "access svn libraries directly" option 
to work is that I'm using 64-bit svn libraries.

Hm. Looks like I will also need the source to libsvnXplus.a. It doesn't seem to 
be included in the 1.2 source.
```

Original comment by `goo...@dwilga.info` on 21 May 2012 at 12:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `chris...@gmail.com` on 24 Aug 2015 at 7:12
- Changed state: **WontFix**
